exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/dml/SelectTests.kt

```diff
@@ -264,13 +264,13 @@ class SelectTests : DatabaseTestsBase() {
         }
     }
 
-    val testDBsSupportingInAnyAllFromTables = TestDB.postgreSQLRelatedDB + TestDB.allH2TestDB
+    private val testDBsSupportingInAnyAllFromTables = TestDB.postgreSQLRelatedDB + TestDB.allH2TestDB
```

Comment:
This should include MySQL8 as well, but tests will fail due to the current issue with gradle task testMySql8. A YT issue has been created to add the dialect to the tests once that issue is resolved.